### DISCUSSION TITLE
Images are referenced as links

### DISCUSSION THREAD

**Discussion by SysCellOne** [akosbalasko/yarle#497] [Q&A]
No images are being displayed in my Markdown files because they are being referenced as follows

`![[./_resources/Matrix.resources/Picture.jpeg]]`

when they should be referenced as

`![image](_resources/Matrix.resources/Picture.jpeg)`

I'm trying to convert to a standard markdown format.... strangely, if I convert to Obsidian MarkDown, the images are displayed.

any suggestions?

**Comment by akosbalasko** [comment]
Hi @SysCellOne ,
it means that your target note taking system interprets images well if they are in ![xxx](file) style. So if you're fine with the Obsidian conversion, use that to get your notes converted, or why do you prefer StandardMD conversion? what is your target application?

**Comment by SysCellOne** [comment]
Thanks for the information.
My intention is to keep the notes Standard MarkDown or Git Flavour MarkDown.
I used Typora and vim+MarkdownPreview.


As a workaround, (the old) Evernote exports  whole notebooks as separate
html files (the new version does not).

I'm using pandoc to convert them to standard MarkDown ..but some html code
still gets through ... the pandoc markdown is not as  clean YARLE markdown
